src/main/java/com/iexec/core/worker/WorkerService.java

```diff
@@ -50,15 +60,47 @@ public WorkerService(WorkerRepository workerRepository,
         this.contextualLockRunner = new ContextualLockRunner<>();
     }
 
+    @PostConstruct
+    void init() {
+        aliveWorkersGauge = Metrics.gauge(METRIC_WORKERS_GAUGE, new AtomicInteger(getAliveWorkers().size()));
+        aliveTotalCpuGauge = Metrics.gauge(METRIC_CPU_TOTAL_GAUGE, new AtomicInteger(getAliveTotalCpu()));
+        aliveAvailableCpuGauge = Metrics.gauge(METRIC_CPU_AVAILABLE_GAUGE, new AtomicInteger(getAliveAvailableCpu()));
+    }
+
+    /**
+     * updateMetrics is used to update all workers metrics
+     */
+    @Scheduled(fixedDelayString = "${cron.metrics.refresh.period}", initialDelayString = "${cron.metrics.refresh.period}")
```

Comment:
Can we have a following PR with an updated `README.md` containing the new parameters ?
It would be nice to add a section like the one recently added on the worker to let users know they can enable prometheus metrics.

Reply:
I think that in a next PR, I would like to read the Prometheus metrics value when calling `/metrics` endpoint.
This would avoid some non-optimized calls on the database for example.